source/features/mark-private-orgs.tsx

```diff
@@ -4,14 +4,14 @@ import features from '../libs/features';
 import * as icons from '../libs/icons';
 import * as api from '../libs/api';
 
-async function init() {
+async function init(): Promise<boolean | void> {
 	const orgs = select.all<HTMLAnchorElement>('.avatar-group-item[data-hovercard-type="organization"]');
 	if (orgs.length === 0) {
 		return false;
 	}
 
-	let publicOrgs = await api.v3(`users/${getUsername()}/orgs`);
-	publicOrgs = publicOrgs.map(orgData => `/${orgData.login}`);
+	let publicOrgs = (await api.v3(`users/${getUsername()}/orgs`))!;
+	publicOrgs = publicOrgs.map((orgData: AnyObject) => `/${orgData!.login}`);
```

Comment:
This will eventually not be of type `AnyObject`. I'm going to see about setting the return type  of `call` as a generic with a default type of `AnyObject`. `call` is used under the hood for `api.v3` and `api.v4`.

Reply:
If it's not required by ts, I prefer `AnyObject` to having to type each API response.